static/sass/_snapcraft_market.scss

```diff
@@ -125,6 +125,14 @@
     .p-inline-list__item {
       line-height: 2.3rem;
     }
+
+    [class*="p-button"]:not([class*="p-button__"]) {
```

Comment:
Thats very specific, where does it come from?

Reply:
Vanilla adds right margin to some elements, and not to others (whether a `button` or `p`, because we use both we get random spacing. This target's both `p-button--neutral` and `p-button--positive`, but excludes `p-button__text` added for the save spinner.

Reply:
Maybe we should make this sticky bar a proper component too?

Reply:
Added a not on that to #2168 as well.